extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/BodyHandlerConfig.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.vertx.http.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Vert.x Web body handler configuration
+ */
+@ConfigGroup
+public class BodyHandlerConfig {
+
+    /**
+     * Whether the files sent using {@code multipart/form-data} will be stored locally.
+     * <p>
+     * If {@code true}, they will be stored in {@code quarkus.http.body-handler.uploads-directory} and will be made
+     * available via {@code io.vertx.ext.web.RoutingContext.fileUploads()}.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean handleFileUploads;
+
+    /**
+     * The directory where the files sent using {@code multipart/form-data} should be stored.
+     * <p>
+     * Either an absolute path or a path relative to the current directory of the application process.
+     */
+    @ConfigItem(defaultValue = "file-uploads")
+    public String uploadsDirectory;
+
+    /**
+     * Whether the form attributes should be added to the request parameters.
+     * <p>
+     * If {@code true}, the form attributes will be added to the request parameters; otherwise the form parameters will
+     * not be added to the request parameters
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean mergeFormAttributes;
+
+    /**
+     * Whether the uploaded files should be removed after serving the request.
+     * <p>
+     * If {@code true} the uploaded files stored in {@code quarkus.http.body-handler.uploads-directory} will be removed
+     * after handling the request. Otherwise the files will be left there forever.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean deleteUploadedFilesOnEnd;
```

Comment:
This one needs to be tested carefully... 
I had issues in a container where it was not behavior properly (not a big deal as the container was getting shutdown)

Reply:
The DisabledUploadsTest https://github.com/quarkusio/quarkus/pull/4118/files#diff-c251801edc089c62bf58421345eefb5dR36 is enough, isn't it?

Reply:
`DisabledUploadsTest` does something else. Sorry for the confusion.

Reply:
Added `DeleteUploadedFilesOnEndTest` in 950b2ad4e552f342f0a4adba99fe99a4ac5bc8de